tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -2146,6 +2146,12 @@ voq/test_fabric_reach.py:
     conditions:
       - "('t2' not in topo_name) or (asic_subtype not in ['broadcom-dnx']) or ('arista_7800' not in platform) or (asic_type in ['cisco-8000'])"
 
+voq/test_voq_fabric_capacity.py:
+  skip:
+    reason: "Skip test_voq_fabric_capacity on unsupported testbed."
+    conditions:
+      - "('t2' not in topo_name) or (asic_subtype not in ['broadcom-dnx']) or ('arista_7800' not in platform) or (asic_type in ['cisco-8000'])"
```

Comment:
This tests should be supported on all DNX platforms? please remove the check `('arista_7800' not in platform)`

Reply:
> This tests should be supported on all DNX platforms? please remove the check `('arista_7800' not in platform)`

I can remove it . I see all other fabric tests are stil running on arista systems, so added before.